graviti/utility/lazy.py

```diff
@@ -233,6 +238,26 @@ def create_list(
         self._lists.append(lazy_list)
         return lazy_list
 
+    def get_lazylists(
+        self, extractors: Mapping[str, Union[_Extractor, _Extractors]]
+    ) -> Dict[str, Any]:
+        """Get lazy lists from extractors.
```

Comment:
`lazy_lists` or `lazylist`
in the docstring, it is `lazy lists` with space as delimiter.